release/cloudbuild-proxy.yaml

```diff
@@ -14,23 +14,24 @@
 # https://cloud.google.com/cloud-build/docs/running-builds/automate-builds
 steps:
 # Set permissions correctly. Not sure why it is necessary, but it is.
```

Comment:
I think this comment might be out of date (not directly related but it's close to the changed code)